### PR TITLE
Guides public copy edits – part 2

### PR DESCRIPTION
**Pull request by rochoa**
This is a continuation of #1088.

cc @makella

### CODE REVIEW

**Comment by makella** [comment]
thank you!

**Comment by VictorVelarde** [comment]
After v0.10.0, these are the main points to be fixed for v1.0.0 release:

- [ ] docs/guides/00-introduction.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/01-getting-started.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/02-using-sources.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/03-style-with-expressions.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/04-data-driven-visualizations-part-1.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/04-data-driven-visualizations-part-2.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/05-add-legends.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/06-aggregations.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/06-zoom-based-styling.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/08-animate-your-visualizations.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/09-interactivity-events.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/10-add-widgets.md
   - [ ] Title: 
   - [ ] Broken links
   - [ ] Broken styles
   - [ ] Remove non primary comments

- [ ] docs/guides/Glossary.md

**Comment by VictorVelarde** [comment]
Great @makella! I have updated our "plan for these 2 days 🔝" and I'll be making fixes and deployments to staging. 

Update both of your branches (at `developers` and `carto-vl`)  when you start today.

File: examples/guides/data-driven-viz-1/step-2.html [hunk 1]
```diff
@@ -2,16 +2,16 @@
 <html>
 
 <head>
-    <title>Implicit casting | CARTO</title>
+    <title>Implicit casts | CARTO</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
     <!-- Include CARTO VL JS -->
-    <script src="../../../dist/carto-vl.min.js"></script>
+    <script src="https://libs.cartocdn.com/carto-vl/branches/971-guides-aggregation/carto-vl.min.js"></script>
```
Comment (davidmanzanares) [inline examples/guides/data-driven-viz-1/step-2.html]: Hey, this is not good, this links to a CVL bundle of a branch, to a patched MGL version and to a carto.com CSS

File: docs/guides/10-add-legends.md [hunk 2]
```diff
@@ -10,36 +10,95 @@ If you completed the XXX guide, the map below will look familiar. In that guide,
...
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Rail accident weather | CARTO</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+    <script src="../../../dist/carto-vl.js"></script>
+    <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
```
Comment (davidmanzanares) [inline docs/guides/10-add-legends.md]: Patched MGL :eyes:

File: docs/guides/04-data-driven-visualizations-part-1.md [hunk 3]
```diff
@@ -119,7 +119,7 @@ There are multiple classification methods aviailable in CARTO VL (quantiles, equ
 - **Data in the viewport**. `viewport*` classifiers will apply the classification only to the features that are in the viewport. This includes filtering by the `filter` styling property and filtering by checking that the feature is within the region covered by the screen at each moment. Changes in the view (map center/map zoom) will trigger an automatic re-computation of the classification.
 
 **Note**
-Learn more about using `global*` and `viewport*` methods in the Aggregations guide.
+Learn more about using `global*` and `viewport*` methods in the Spatial aggregations guide.
```
Comment (davidmanzanares) [inline docs/guides/04-data-driven-visualizations-part-1.md]: Not correct: `global*` agg is not spatial

File: docs/guides/04-data-driven-visualizations-part-1.md [hunk 4]
```diff
@@ -141,14 +141,14 @@ Do you see how `viewport*` classifiers are dynamic and change the results accord
 
 <div class="example-map">
     <iframe
-        id="population-density-classified"
-        src="/developers/carto-vl/examples/maps/guides/ramp/population-density-classified.html"
+        id="guides-data-driven-1-step-4"
```
Comment (davidmanzanares) [inline docs/guides/04-data-driven-visualizations-part-1.md]: I don't agree much with this change as you know :man_shrugging:

File: docs/guides/05-data-driven-visualizations-part-2.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+- `ramp($dn, cb_blues)`
+
+You can find scheme names for CARTOColors [here](https://carto.com/carto-colors/) and for ColorBrewer [here](http://colorbrewer2.org).
+
+Use the map below, to switch between various color expressions and color schemes:
+
+```CARTO_VL_Viz
+// Style 1: apply colors from a color list of named colors
+
+color: ramp($dn, [green, yellow, red])
```
Comment (davidmanzanares) [inline docs/guides/05-data-driven-visualizations-part-2.md]: The original styles showed some things these examples don't: ability to mix different forms of colors (like named colors with `rgb()`), hex colors and `opacity`

File: docs/guides/06-spatial-aggregations.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-## Aggregating data
+## Spatial aggregations
```
Comment (davidmanzanares) [inline docs/guides/06-spatial-aggregations.md]: Not true, `global*` is not a spatial aggregation

File: docs/guides/06-spatial-aggregations.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
...
 
 In this guide you will explore a set of functions called **aggregations**, that allow you to extract different values from your datasets by considering either the whole set of features (*global aggregations*), just the ones displayed in the current map extent (*viewport aggregations*) or those derived from spatial aggregations (*cluster aggregations*).
```
Comment (davidmanzanares) [inline docs/guides/06-spatial-aggregations.md]: the title is `Spatial aggregations` but here three different types of aggs are mentioned, only one of them is called "spatial".

I think that `viewport*` agg is technically spatial, although I think that it's behavior is more similar to `global*` aggs and therefore I wouldn't separate `global*` from `viewport*` + `cluster*`

File: examples/guides/data-driven-viz-1/step-2.html (hunk 1 above)
Comment (rochoa) [reply examples/guides/data-driven-viz-1/step-2.html]: 🤔

I already fixed similar cases when I merged master into this branch in the previous PR, see https://github.com/CartoDB/carto-vl/commit/2ea69c20663953b27e3a0b5c9effd519bb170e2e.

File: docs/guides/07-zoom-based-styles.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-## Zoom-based Styling
+## Zoom-based styles
```
Comment (davidmanzanares) [inline docs/guides/07-zoom-based-styles.md]: I don't understand the purpose of this change. `Zoom-based styles` sounds to me like if there were a fixed number of zoom-based styles.

File: docs/guides/07-zoom-based-styles.md [hunk 9]
```diff
@@ -44,23 +43,98 @@ After you explore each one independently, you will bring them together to create
 ### Getting started
 To get started, copy and paste the code below into your favorite text editor and save it to a file `vancouver-trees.html`.
 
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+
```
Comment (davidmanzanares) [inline docs/guides/07-zoom-based-styles.md]: I would get rid of unimportant blank lines too

File: docs/guides/07-zoom-based-styles.md [hunk 10]
```diff
@@ -44,23 +43,98 @@ After you explore each one independently, you will bring them together to create
...
+    <script src="../../../dist/carto-vl.js"></script>
+
+    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.50.0/mapbox-gl.js'></script>
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.50.0/mapbox-gl.css' rel='stylesheet' />
+
+    <link rel="stylesheet" type="text/css" href="../../style.css">
+    <style>
+        <!-- Zoom level indicator style -->
```
Comment (davidmanzanares) [inline docs/guides/07-zoom-based-styles.md]: I don't think we should focus on the zoom level indicator, that is useful to understand and see how the zoom-based styling works, but I don't expect to see it on final real apps

File: docs/guides/07-zoom-based-styles.md [hunk 11]
```diff
@@ -44,23 +43,98 @@ After you explore each one independently, you will bring them together to create
...
+            font-size: 16px;
+        }
+    </style>
+</head>
+
+<body>
+
+    <div id="map"></div>
+
+    <!-- Zoom level indicator container -->
```
Comment (davidmanzanares) [inline docs/guides/07-zoom-based-styles.md]: Same here about the comment and about the blank line

File: docs/guides/07-zoom-based-styles.md [hunk 12]
```diff
@@ -44,23 +43,98 @@ After you explore each one independently, you will bring them together to create
...
+            style: carto.basemaps.darkmatter,
+            center: [-123.098599, 49.240685],
+            zoom: 11
+        });
+
+        const nav = new mapboxgl.NavigationControl();
+        map.addControl(nav, 'top-left');
+
+        // Zoom level indicator
+        function updateZoom() {
```
Comment (davidmanzanares) [inline docs/guides/07-zoom-based-styles.md]: In fact, I just saw that the whole purpose of this example is the indicator. It doesn't make sense to remove the comments then, but I think we should waste so much space in putting **inline** an example that only shows a code useful for debugging.

File: docs/guides/09-add-interactivity-and-events.md [hunk 13]
```diff
@@ -1,11 +1,11 @@
-## Interactivity and events
+## Add interactivity and events
```
Comment (davidmanzanares) [inline docs/guides/09-add-interactivity-and-events.md]: `Add` here for me is just extra noise

File: docs/guides/09-add-interactivity-and-events.md [hunk 14]
```diff
@@ -321,10 +319,6 @@ Here it is the full example:
         };
         map.on('move', displayCenter);
 
-
-        //** CARTO VL functionality begins here **//
-
-
         // LAYER EVENTS & VARIABLES
         // Add layer as usual
```
Comment (davidmanzanares) [inline docs/guides/09-add-interactivity-and-events.md]: // LAYER EVENTS & VARIABLES	    
        // Add layer as usual	

seems like bad comments to me

File: examples/guides/add-legends/step-9.html [hunk 15]
```diff
@@ -0,0 +1,91 @@
+!DOCTYPE html>
+<html>
+
+<head>
+  <script src="https://libs.cartocdn.com/carto-vl/v0.10.0/carto-vl.min.js"></script>
```
Comment (davidmanzanares) [inline examples/guides/add-legends/step-9.html]: links to CVL bundle and CSS are not ok

File: examples/guides/add-legends/step-9.html [hunk 16]
```diff
@@ -0,0 +1,91 @@
...
+    });
+    map.touchZoomRotate.disableRotation();
+
+    // Authenticate the client
+    carto.setDefaultAuth({
+      username: 'cartovl',
+      apiKey: 'default_public'
+    });
+
+    // Define the source
```
Comment (davidmanzanares) [inline examples/guides/add-legends/step-9.html]: comments

File: examples/guides/data-driven-viz-2/step-1.html [hunk 17]
```diff
@@ -106,9 +101,8 @@ <h2>'ramp($dn, [transparent, gray, yellow])'</h2>
                 colorLegendList +=
                     `<li><span class="point-mark" style="background-color:${colorHex};"></span> <span>${key}</span></li>\n`;
             });
-            // Update the information
             document.getElementById('content').innerHTML = colorLegendList;
-            document.getElementById('content-title').innerHTML = `Legend of ${colorLegend.name}`;
+            document.getElementById('content-title').innerHTML = `Legend`;
```
Comment (davidmanzanares) [inline examples/guides/data-driven-viz-2/step-1.html]: Why this change?

File: examples/guides/data-driven-viz-2/step-2.html [hunk 18]
```diff
@@ -122,7 +118,7 @@ <h2>ramp(globalEqIntervals($total_damage, 3), [1, 25])</h2>
 
                 if (color) {
                     colorLegendList +=
-                        `<li><span class="point-mark" style="background-color:${color}; border: 1px solid black;"></span><span>${legend.key.replace('CARTO_VL_OTHERS', 'Rest of types')}</span></li>\n`;
+                        `<li><span class="point-mark" style="background-color:${color}; border: 1px solid black;"></span><span>${legend.key.replace('CARTO_VL_OTHERS', 'All other causes')}</span></li>\n`;
```
Comment (davidmanzanares) [inline examples/guides/data-driven-viz-2/step-2.html]: The style is by accident type, not the accident cause ( see referenced column)

File: examples/guides/data-driven-viz-2/step-3.html [hunk 19]
```diff
@@ -0,0 +1,105 @@
...
+
+  <script>
+    const map = new mapboxgl.Map({
+      container: 'map',
+      style: carto.basemaps.darkmatter,
+      center: [-73.954323, 40.711743],
+      zoom: 15.7,
+
+    });
+    map.touchZoomRotate.disableRotation();
```
Comment (davidmanzanares) [inline examples/guides/data-driven-viz-2/step-3.html]: disablerotation

File: examples/guides/data-driven-viz-2/step-3.html [hunk 20]
```diff
@@ -0,0 +1,105 @@
...
+   
+    carto.setDefaultAuth({
+      username: 'cartovl',
+      apiKey: 'default_public'
+    });
+    
+    const source = new carto.source.Dataset('table_311_complaints');
+    
+    // Define Viz object and custom style
```
Comment (davidmanzanares) [inline examples/guides/data-driven-viz-2/step-3.html]: comments

File: examples/guides/data-driven-viz-2/step-4.html [hunk 21]
```diff
@@ -0,0 +1,105 @@
...
+    map.touchZoomRotate.disableRotation();
+   
+    carto.setDefaultAuth({
+      username: 'cartovl',
+      apiKey: 'default_public'
+    });
+    
+    const source = new carto.source.Dataset('table_311_complaints');
+    
+    // Define Viz object and custom style
```
Comment (davidmanzanares) [inline examples/guides/data-driven-viz-2/step-4.html]: comments

File: examples/misc/legends/poly-legend-ma.html [hunk 22]
```diff
@@ -74,8 +74,8 @@ <h1>Percent Master's Degree</h1>
 
       colorLegend.data.forEach((legend) => {
         const colorHex = rgbToHex(legend.value);
-        const keyMin = legend.key[0].toFixed(2).replace('Infinity', '<');
-        const keyMax = legend.key[1].toFixed(2).replace('Infinity', '>');
+        const keyMin = legend.key[0].toFixed(2).replace('Infinity', '∞');
```
Comment (davidmanzanares) [inline examples/misc/legends/poly-legend-ma.html]: I don't understand the reason for this change. I thought @makella actually complained about showing `Infinity`, but the original code replaced it to be more user friendly. :confused:

File: examples/guides/data-driven-viz-1/step-2.html (hunk 1 above)
Comment (VictorVelarde) [reply examples/guides/data-driven-viz-1/step-2.html]: Thx david, I'll fix them. I'm checking all the links

File: docs/guides/10-add-legends.md (hunk 2 above)
Comment (VictorVelarde) [reply docs/guides/10-add-legends.md]: 👍 good catch.  I have replaced all v0.48 references with 0.50 and also fixed some bad hardcoded urls for these guides: '10-add-legends' and '04-data-driven-visualizations-part-1'

File: docs/guides/04-data-driven-visualizations-part-1.md (hunk 4 above)
Comment (VictorVelarde) [reply docs/guides/04-data-driven-visualizations-part-1.md]: Yeah, but I'm afraid that another refactoring in that topic will not fit our current timing. Maybe you can create an issue with this for it to be reconsidered.

File: docs/guides/07-zoom-based-styles.md (hunk 10 above)
Comment (VictorVelarde) [reply docs/guides/07-zoom-based-styles.md]: I have removed that comment

File: docs/guides/04-data-driven-visualizations-part-1.md (hunk 4 above)
Comment (davidmanzanares) [reply docs/guides/04-data-driven-visualizations-part-1.md]: Yes, I totally agree that now is not the time

File: docs/guides/07-zoom-based-styles.md (hunk 11 above)
Comment (VictorVelarde) [reply docs/guides/07-zoom-based-styles.md]: I have removed several comments and blank lines here

File: docs/guides/09-add-interactivity-and-events.md (hunk 14 above)
Comment (VictorVelarde) [reply docs/guides/09-add-interactivity-and-events.md]: 👍 removed!

File: examples/guides/add-legends/step-9.html (hunk 15 above)
Comment (VictorVelarde) [reply examples/guides/add-legends/step-9.html]: Fixed

File: examples/guides/data-driven-viz-2/step-3.html (hunk 19 above)
Comment (VictorVelarde) [reply examples/guides/data-driven-viz-2/step-3.html]: Removed!

File: examples/guides/data-driven-viz-2/step-4.html [hunk 23]
```diff
@@ -0,0 +1,105 @@
...
+
+  <script>
+    const map = new mapboxgl.Map({
+      container: 'map',
+      style: carto.basemaps.darkmatter,
+      center: [-73.954323, 40.711743],
+      zoom: 15.7,
+
+    });
+    map.touchZoomRotate.disableRotation();
```
[the remark this answers is not in the document]
Comment (VictorVelarde) [reply examples/guides/data-driven-viz-2/step-4.html]: I have removed all references to this old disable, from here and other examples

File: examples/guides/data-driven-viz-2/step-4.html (hunk 21 above)
Comment (VictorVelarde) [reply examples/guides/data-driven-viz-2/step-4.html]: I'm removing 'spurious comments' in several guides and examples, but a further review in the future might be needed with this topic

File: docs/guides/06-spatial-aggregations.md (hunk 7 above)
Comment (VictorVelarde) [reply docs/guides/06-spatial-aggregations.md]: I aggree

File: examples/misc/legends/poly-legend-ma.html (hunk 22 above)
Comment (davidmanzanares) [reply examples/misc/legends/poly-legend-ma.html]: Sorry, I didn't mean to be rude, my English fault, sorry.

I answered to the related issue too https://github.com/CartoDB/carto-vl/issues/1110#issuecomment-436231428

File: docs/guides/06-spatial-aggregations.md (hunk 6 above)
Comment (VictorVelarde) [reply docs/guides/06-spatial-aggregations.md]: 🔝

File: examples/guides/data-driven-viz-2/step-2.html (hunk 18 above)
Comment (makella) [reply examples/guides/data-driven-viz-2/step-2.html]: ok, feel free to change "Rest of Types" didn't make sense to me since there wasn't a description of what the map was showing

File: examples/misc/legends/poly-legend-ma.html (hunk 22 above)
Comment (makella) [reply examples/misc/legends/poly-legend-ma.html]: i couldn't get rid of the `-` so i reverted back to the infinity sign

File: docs/guides/05-data-driven-visualizations-part-2.md (hunk 5 above)
Comment (makella) [reply docs/guides/05-data-driven-visualizations-part-2.md]: @davidmanzanares I didn't actually remove any of what you had... there were the series of examples that were in the text (and are kept like that):
```
- `ramp($dn, [rgb(200,220,222), rgba(200,120,22, 0.8)])`
- `ramp($dn, [hsv(0,1,1), hsv(0.5,1,1)]`
- `ramp($dn, [#00F, #F00])`
- `ramp($dn, [blue, #F00])`
- `ramp($dn, [opacity(blue, 0.4), opacity( #F00, 0.6),])`
```
from that section, I see this one: `ramp($dn, [opacity(blue, 0.4), opacity( #F00, 0.6),])`

and for the map, i didn't add, remove, or modify any of the styles i just extracted the radio button text to the code block above... the map. style 2 uses tranparent + named colors

```
// Style 1: apply colors from a color list of named colors

color: ramp($dn, [green, yellow, red])

// Style 2: apply colors from a color list including "transparent"

color: ramp($dn, [transparent, gray, yellow])

// Style 3: apply colors from a color list with HSV color constructors

color: ramp($dn, [hsv(0,1,1), hsv(0.5,1,1)])

// Style 4: apply color using the CARTOColor scheme "temps"

color: ramp($dn, temps)
```
Maybe I accidentally got rid of an example (?) anyway, feel free to add back... wasn't intentional.

Comment (davidmanzanares) [reply docs/guides/05-data-driven-visualizations-part-2.md]: Ah, sorry I was trying to compare this to the previous version and I compared the styles here with the styles given a few lines before (which I didn't see in this version, but, as you point out, they are there), Sorry for the noise, everything is good :slightly_smiling_face:
